attribution_scopes.md

```diff
@@ -0,0 +1,236 @@
+
+```jsonc
+{
+  ..., //existing fields
+
+  // Optional
+  // Represents a list of attribution scopes for a particular trigger.
+  // Triggers will only match sources whose attribution_scopes contains at least one of the trigger’s
+  // attribution_scopes, if specified.
+  // Attribution scope values must be strings. Each string has a maximum length of 50.
+  // Each list has a maximum length of 20.
+  // Defaults to the empty list if omitted.
+  "attribution_scopes": <list of strings>,
+
+}
+```
+
+Only sources whose `attribution_scopes` contains at least one of the trigger’s `attribution_scopes` will be considered for attribution. Note that this filtering takes place before attributing a trigger to a particular source.
+
+If there are multiple sources whose `attribution_scopes` contains at least one of the trigger’s `attribution_scopes`, then attribution will take place among the sources within the particular `attribution_scopes` according to current [Attribution Reporting API logic](https://github.com/WICG/attribution-reporting-api/blob/main/EVENT.md#trigger-attribution-algorithm).
+
+If the trigger registration does not specify an `attribution_scopes` then all sources are considered for attribution.
+
+Once an `attribution_scope_limit` is set, the last K values (i.e. where K = `attribution_scope_limit`) of `attribution_scopes` will be considered the final set of `attribution_scopes` values and any source with additional `attribution_scopes` values will be treated as if the attribution scopes were empty.
+
+
+If a source registration is specified with a configuration that has a higher number of potential event states than the most recent `max_event_states` for the same reporting origin, then the source will be rejected and the registration will fail. Additionally, if the `max_event_states` field is changed in a future source registration, then all other previous pending source registrations with a different `max_event_states` will be ignored in subsequent attribution report generation flows, but will still count towards rate limits. 
+
+
+### Updating attribution scope values
+
+An API caller may want to update the value of `attribution_scope_limit` for certain registrations or at a certain time. For example, when an advertiser starts a new campaign the API caller may want to increase the value of `attribution_scope_limit` to account for this new campaign.
+
+The `attribution_scope_limit` value can be updated during source registration at any time. However, any pending sources (previously registered) that have been specified with an `attribution_scope_limit` less than the current source registration’s `attribution_scope_limit`  will be deleted.
+
+
+### Deletion logic
+
+If a source is attributed to the current trigger, then all other eligible sources considered for attribution (across all `attribution_scopes`) will be deleted.
```

Comment:
I think we usually say that a trigger is attributed to a source, not the other way around.